source/documentation/cli/dart-sass.md

```diff
@@ -0,0 +1,582 @@
+does not watch additional directories based on a file's `@import`/`@use`/
+`@forward` rules.
+
+```shellsession
+$ sass --watch themes:public/css
+Compiled themes/light.scss to public/css/light.css.
+
+​# Then when you edit themes/dark.scss...
+Compiled themes/dark.scss to public/css/dark.css.
+```
+
+#### `--poll`
+
+{% compatibility '1.8.0' %}{% endcompatibility %}
+
+This flag, which may only be passed along with `--watch`, tells Sass to manually
+check for changes to the source files every so often instead of relying on the
+operating system to notify it when something changes. This may be necessary if
+you're editing Sass on a remote drive where the operating system's notification
+system doesn't work.
+
+```shellsession
+$ sass --watch --poll themes:public/css
+Compiled themes/light.scss to public/css/light.css.
+
+​# Then when you edit themes/dark.scss...
+Compiled themes/dark.scss to public/css/dark.css.
+```
+
+#### `--stop-on-error`
+
+{% compatibility '1.8.0' %}{% endcompatibility %}
+
+This flag tells Sass to stop compiling immediately when an error is detected,
+rather than trying to compile other Sass files that may not contain errors. It's
+mostly useful in [many-to-many mode][].
+
+```shellsession
+$ sass --stop-on-error themes:public/css
+Error: Expected expression.
+   ╷
+42 │ h1 {font-face: }
+   │                ^
+   ╵
+  themes/light.scss 42:16  root stylesheet
+```
+
+#### `--interactive`
+
+{% compatibility '1.5.0' %}{% endcompatibility %}
+
+This flag (abbreviated `-i`) tells Sass to run in interactive mode, where you
+can write [SassScript expressions][] and see their results. Interactive mode
+also supports [variables][] and [`@use` rules][].
+
+[SassScript expressions]: /documentation/syntax/structure#expressions
+[variables]: /documentation/variables
+[`@use` rules]: /documentation/at-rules/use
+
+```shellsession
+$ sass --interactive
+>> 1px + 1in
+97px
+>> @use "sass:map"
+>> $map: ("width": 100px, "height": 70px)
+("width": 100px, "height": 70px)
+>> map.get($map, "width")
+100px
+```
+
+#### `--color`
+
+This flag (abbreviated `-c`) tells Sass to emit [terminal colors][], and the
+inverse `--no-color` tells it not to emit colors. By default, it will emit
+colors if it looks like it's being run on a terminal that supports them.
+
+[terminal colors]: https://en.wikipedia.org/wiki/ANSI_escape_code#Colors
+
+<pre><code>$ sass --color style.scss style.css
+Error: Incompatible units em and px.
+  <span style="color: blue">╷</span>
+<span style="color: blue">1 │</span> $width: <span style="color: crimson">15px + 2em</span>
+  <span style="color: blue">│</span>         <span style="color: crimson">^^^^^^^^^^</span>
+  <span style="color: blue">╵</span>
+  style.scss 1:9  root stylesheet
+
+$ sass --no-color style.scss style.css
+Error: Incompatible units em and px.
+  ╷
+1 │ $width: 15px + 2em
+  │         ^^^^^^^^^^
+  ╵
+  style.scss 1:9  root stylesheet</code></pre>
+
```

Comment:
These manual `<pre><code>` blocks don't have the `language` class like the others. Should we add that here? I've seen the different bg color in a few places (first two code blocks on this page).

Reply:
@stacyk I adjusted these in https://github.com/oddbird/sass-site/pull/42/commits/b47c9c59ef4080455a020c421c36b630ba2f7cd4, but there are a number of plain `<pre><code>` (or rather, plain ```) blocks in blog posts that I did not fix. Most of these have no syntax highlighting anyway, and it seems a bit weird to me to add `language-plaintext` everywhere just to get the background color. Could we more globally apply the background color to `<pre><code>` instead?

Reply:
Sounds good -- feel free to try that. See https://sass-site-oddbird.netlify.app/blog/security-alert-tar-permissions/ for an example with blocks that have the wrong bg color.

Reply:
@jgerigmeyer I updated the Sass to keep those bg colors consistent.